site/content/api/Tenant-API.md

```diff
@@ -277,32 +332,62 @@ Devices belonging to the tenant can connect to Hono via the rest-adapter only an
     {
       "type": "hono-http",
       "enabled": true,
-      "device-authentication-required": true
+      "device-authentication-required": true,
+      "deployment": {
+        "maxInstances": 4
+      }
     }
   ]
 }
 ~~~
 
-In the following example the tenant is allowed to use **all** adapters, as the `adapters` property is omitted in the tenant configuration:
+In the following example the tenant is allowed to use **all** adapters, as the `adapters` property is omitted in the tenant configuration (note that the payload also contains a custom property *plan* which might be used to indicate the customer's service level):
```

Comment:
`...tenant configuration.
Note that ... :`

That is how it is written below  at same place, and I think to begin with a new sentence makes it easier to read.